app/views/account/weeklyRenew.scala.html

```diff
@@ -48,47 +48,20 @@ <h3 class="mma-section__header">
             </section>
 
             <section class="mma-section">
-                <h3 class="mma-section__header">
-                    Bill to country
-                </h3>
-                @billToCountry
-            </section>
-
-            <section class="mma-section">
-                <h3 class="mma-section__header">
-                    Email address
-                </h3>
-                @contact.email
-                @helper.CSRF.formField
-            </section>
+                <h3 class="mma-section__header">Renew your subscription</h3>
 
-                @if(subscription.renewable){
+                <p>
+                @if(subscription.termEndDate.isAfter(LocalDate.now)) {
+                    Your current subscription is due to expire on @prettyDate(subscription.termEndDate).
+                } else {
+                    Your subscription expired on @prettyDate(subscription.termEndDate).
+                }
+                Please complete the form below to renew your subscription. You can find your Promo Code on any correspondence from us.
```

Comment:
do we always send a promo code? not sure if this would be on all the letters anyway?  maybe no need to mention promo codes separately